uk_election_ids/election_ids.py

```diff
@@ -409,3 +367,57 @@ def __repr__(self):
 
     def __eq__(self, other):
         return type(other) == IdBuilder and self.__dict__ == other.__dict__
+
+    @classmethod
+    def parse_from_id(cls, election_id: str) -> "IdBuilder":
```

Comment:
I'd probably just make this `IdBuilder.from_id()`, similar to Django's `Manager.from_queryset()` or python's `datetime.fromtimestamp()`, etc.